src/osd/CMakeLists.txt

```diff
@@ -57,3 +57,15 @@ endif()
 if(WITH_OSD_INSTRUMENT_FUNCTIONS)
   add_dependencies(osd cyg_profile_tp)
 endif()
+
+add_dependencies(osd cls_journal cls_hello cls_lock cls_log cls_numops
+  cls_refcount cls_timeindex cls_user cls_version cls_cas)
+if (WITH_CEPHFS)
```

Comment:
nit, might want to remove the space after `if` to be more consistent.

Reply:
Ack, thanks